network-autoconnections-httpks.ks.in

```diff
@@ -27,8 +27,12 @@ shutdown
 
 # We will copy /root/RESULT to /mnt/sysimage/root/RESULT at the beginning of the %post (--nochroot) checks
 @KSINCLUDE@ post-lib-network.sh
-
-check_ifcfg_exists @KSTEST_NETDEV1@ no
+if [[ "@KSTEST_OS_NAME@" == "rhel" ]]; then
+    # No NM in initramfs in RHEL8.2 => ifcfg file is created by dracut network module
```

Comment:
It will have to be reverted in RHEL 8.3, right? Is it worth it?

Reply:
I'd prefer to have it fixed for the rest of 8.2 phase. The test might catch other issues (eg changes in dracut, NM).